### PR TITLE
added requirements.txt and created Readme.md

### PR DESCRIPTION
**Pull request by iamtalhaasghar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Python-Vcf-Contacts-Duplicates-Merger
+Read a contacts file (.vcf). Compare datasets with identical names. Find unique data fields for each name. If same field with different content found, add another entry to the field. Create merged dataset for each name with all unique fields. Create new vcf with merged contacts (only one vCard per name).
+
+# How to Use
+1. Download or clone the source code
+2. Open terminal inside the downloaded folder
+3. `pip install -r requirements.txt`
+4. Change `input_file_path` variable in `line 4` of file `vcf-duplicates-merger.py` to point the target vcf file.
+5. `python cf-duplicates-merger.py`
```
Comment (freprec) [inline README.md]: Thank you :)
```suggestion
5. `python vcf-duplicates-merger.py`
```

Comment (iamtalhaasghar) [reply README.md]: Fixed typo in commit #5386ebd6dc6be5bdc9becdcd2873c27d697ffed5

Comment (iamtalhaasghar) [reply README.md]: Thank your for your review. I have just fixed the typo mistake you mentioned

**Comment by freprec** [comment]
Thank you for your contribution @iamtalhaasghar!

**Comment by iamtalhaasghar** [comment]
This script helped me to remove duplicate contacts from 5000+ contacts and reduced them to 400+ contacts.

Thanks to you @freprec man. Your script saved me from a huge headache.

**Pull request merged**